package.json

```diff
@@ -20,7 +25,6 @@
   },
   "homepage": "https://github.com/frandiox/vite-ssr#readme",
   "bin": "./bin/cli.js",
-  "types": "@types/index.d.ts",
```

Comment:
do we still need this field if we generate them in dist?

Reply:
I think we don't 🤔